fritzconnection/lib/fritzhosts.py

```diff
@@ -148,6 +148,19 @@ def set_host_name(self, mac_address, name):
         }
         self._action('X_AVM-DE_SetHostNameByMACAddress', arguments=args)
 
+    def get_host_name(self, mac_address):
+        """
+        Returns a String with the host_name of the device with the given mac_address
+        """
+        host = list
```

Comment:
This line is not necessary. It just binds the list callable to a new label never used later on.